lib/oelite/log.py

```diff
@@ -1,22 +0,0 @@
-import logging
```

Comment:
Hmm... What do you think about the idea behind this module?  Maybe we should start cleaning up debug and logging instead of just removing the only (although completely stalled) attempt at doing something about it.